c++/src/kj/important_qnx_diffs

```diff
@@ -0,0 +1,726 @@
+22c22,25
```

Comment:
This is my attempt to make it easier to see the diffs between `filesystem-disk-unix.c++` and `filesystem-disk-qnx.c++`, but I don't think it worked.